gui/gui.go

```diff
@@ -819,9 +817,15 @@ func (gui *GUI) Screenshot(path string) {
 	if err != nil {
 		panic(err)
 	}
-	file, _ := os.Create(path)
+	file, err := os.Create(path)
+	if err != nil {
+		panic(err)
+	}
 	defer file.Close()
-	png.Encode(file, img)
+	err = png.Encode(file, img)
+	if err != nil {
+		panic(err)
```

Comment:
Maybe just log instead of panicking? A failed screenshot isn't fatal to Aminal and there are plenty of reasons why writing out the screenshot might fail.

Reply:
That method is used only when unit testing, so there is no much difference in this case. By the way, there are plenty of places where `panic()` is used, so I'd refactor them as a separate PR.

Reply:
Finally decided to move the error handling out of the `gui.Screenshot()` function, so that `panic()` is now called when testing.

Reply:
done in commit https://github.com/liamg/aminal/pull/262/commits/29e2f6861a25f160506bded5cce4e48e6e836a80